doc/lua_api.txt

```diff
@@ -7859,6 +7862,17 @@ Used by `minetest.add_particle`.
         glow = 0
         -- Optional, specify particle self-luminescence in darkness.
         -- Values 0-14.
+
+        node = {name = "ignore", param1 = 0, param2 = 0},
+        -- Optional, if specified the particle will have the same appearance as
+        -- node dig particles for the given node.
```

Comment:
```
-- `param1` is ignored
```

Reply:
Removed it from the shown table instead.